test/conftest.py

```diff
@@ -437,7 +437,7 @@ def init_database():
     # Create the database and the database table
 
     with current_app.app_context():
-        db.create_all()  # todo- use manage.py
+        db.create_all()
```

Comment:
Removing as todo is pretty stale. It seems fine to just use `db.create_all()` here. I briefly looked into using `subprocess.Popen(['./app/manage.py', 'db upgrade'], shell=True)` but ran into import errors. Open to alternative solutions here.